publish_period: 0.01
max_expected_latency: 0.1

command_in_type: "unitless"
scale:
  linear: 0.2
  angular: 0.8
  joint: 0.5

command_out_type: trajectory_msgs/JointTrajectory

publish_joint_positions: true
publish_joint_velocities: true
publish_joint_accelerations: false

use_smoothing: true
smoothing_filter_plugin_name: "online_signal_smoothing::AccelerationLimitedPlugin"

is_primary_planning_scene_monitor: false
check_octomap_collisions: false

move_group_name: gen3_7dof

lower_singularity_threshold: 10.0
hard_stop_singularity_threshold: 30.0
leaving_singularity_threshold_multiplier: 2.0

joint_limit_margins: [0.12]

cartesian_command_in_topic: /tidybot/arm/twist
joint_command_in_topic: /tidybot/arm/joint_jog
status_topic: /tidybot/arm/status
command_out_topic: /gen3_7dof_controller/joint_trajectory

check_collisions: false
collision_check_rate: 10.0
self_collision_proximity_threshold: 0.01
scene_collision_proximity_threshold: 0.02